scripts/functions/help_system

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+function print_arrays()
+{
+    #### This function uses indirect expansion (pass variable name, not the value)
+    ##
+    ## Syntax:  print_arrays args args_about
```

Comment:
why print_arrays ? this feels like print_options

Reply:
Sure I can change the name easily enough

Reply:
By the way, I wrote this a long time ago. Nothing stolen! I got tired of using other peoples bash arg parsers which were making heavy use of other tools to get justification correct (awk, sed, grep, etc). I've been looking forward to actually using it somehow!

https://gist.github.com/milljm/a9d7336b649a7341df2d2fd262820ac5